.github/workflows/main.yml

```diff
@@ -7,148 +7,208 @@ on:
 
 env:
   NB_SERVICES: ${{ secrets.NB_SERVICES }}
-  OPTIMIZER_ORTOOLS_VERSION: ${{ secrets.OPTIMIZER_ORTOOLS_VERSION }}
   PROJECT: optimizer
-  REGISTRY: ${{ secrets.REGISTRY }}
-  VROOM_VERSION: ${{ secrets.VROOM_VERSION }}
 
 jobs:
-  test_basis:
+  build:
+    if: github.event.pull_request.draft == false
```

Comment:
Is there a variable that distinguishes manually launched workflow? If so, we can skip drafts unless the workflow is launched manually. This would prevent an unnecessary un-draft/draft cycle.

https://github.com/asyncapi/.github/issues/13#issue-748881144 if this link is correct, we need to add `ready_for_review` to the `types` parameter; otherwise, even marking the PR as ready is not enough to launch the tests and one needs to do un-draft+closePR+openPR to launch the tests.

Reply:
Indeed adding *ready_for_review* is a plus as it will automatically trigger the pipeline when un-drafting the PR. 

There is no variable that distinguishes manually launched workflow as triggering the pipeline manually means you want to run it.